internal/controllers/port/tests/port-create-sriov/00-port.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: openstack.k-orc.cloud/v1alpha1
+kind: Network
+metadata:
+  name: port-create-sriov
```

Comment:
Either create the `port-create-sriov` network and port in `00-prerequisites.yaml` or in `00-port.yaml`, but not in both.

Reply:
Done